UnityAds/src/main/java/com/mopub/mobileads/UnityInterstitial.java

```diff
@@ -91,6 +92,7 @@ public void onInitializationFailed(UnityAds.UnityAdsInitializationError unityAds
     private IUnityAdsLoadListener mUnityLoadListener = new IUnityAdsLoadListener() {
         @Override
         public void onUnityAdsAdLoaded(String placementId) {
+            mLoadedPlacementId = placementId;
```

Comment:
Looking at this setup - Does this mean that we can only load a single placement ID at a time in mopub?

Reply:
Edit: Yes this is exactly the case because this is a native Ad Unit, and should only have a single placement ID associated 👍